x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/permissions.tsx

```diff
@@ -14,30 +14,28 @@ export const FleetPermissionsCallout = () => {
   return (
     <EuiCallOut title={NEED_PERMISSIONS_PRIVATE_LOCATIONS} color="warning" iconType="help">
       <p>{NEED_PRIVATE_LOCATIONS_PERMISSION}</p>
+      <p>
+        <FormattedMessage
+          id="xpack.synthetics.privateLocations.needFleetPermission.description"
+          defaultMessage="Once you have an agent policy created, you can add private location and manage monitors with just Synthetics app permissions."
```

Comment:
```suggestion
          defaultMessage="Once there is an agent policy available, you'll be able to add manage private locations and monitors with the regular Synthetics app privileges."
```
I believe we use the word 'privilege' for anything permission-related in Kibana. Feel free to correct me if that's not the case.